src/withPoints/pgr_withPoints.cpp

```diff
@@ -27,7 +27,7 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
  ********************************************************************PGR-GNU*/
 
-
+#include "c_types/point_on_edge_t.h"
```

Comment:
Same problem as above

Reply:
It fails to build if I move `#include "c_types/point_on_edge_t.h"` below `#include "withPoints/pgr_withPoints.hpp"`